Sources/Calendar.php

```diff
@@ -365,6 +423,55 @@ function CalendarPost()
 	$context['linktree'][] = array(
 		'name' => $context['page_title'],
 	);
+
+	loadCSSFile('jquery-ui.datepicker.css', array('defer' => false), 'smf_datepicker');
+	loadCSSFile('jquery.timepicker.css', array('defer' => false), 'smf_timepicker');
+	loadJavascriptFile('jquery-ui.datepicker.min.js', array('defer' => true), 'smf_datepicker');
+	loadJavascriptFile('jquery.timepicker.min.js', array('defer' => true), 'smf_timepicker');
+	loadJavascriptFile('datepair.min.js', array('defer' => true), 'smf_datepair');
+	addInlineJavascript('
+	$("#allday").click(function(){   
+		$("#start_time").attr("disabled", this.checked);
+		$("#end_time").attr("disabled", this.checked);
+		$("#tz").attr("disabled", this.checked);
+	});
+	$( "#event_time_input .date_input" ).datepicker({
+		dateFormat: "yy-mm-dd",
```

Comment:
`yy-mm-dd` is that default date format for SMF now...? I would think it should be default of `mm-dd-yyyy`.